### PR TITLE
Blog_app:Views

### PR DESCRIPTION
**Pull request by ethred**
I implemented the following features in this branch:- 

- Implemented the design from the sneak peek wireframes.
- Use methods that i have created in the Blog App - processing data in models

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1,12 +1,22 @@
...
+          <li id="comment">
+            <p><%= comment.user.name %>: <%= comment.text %></p>
+          </li>
+        <% end %>
+      </ul>
+    </li>
+  <% end %>
+</ul>
+
+<%= will_paginate %>
```
Comment (VICTOR-CHUKWU) [inline app/views/posts/index.html.erb:22]: - [ ] when I navigate to post index I get an error of unknown method paginate, kindly fix this
image

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @ethred  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- Views look like given wireframe


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ethred** [comment]
hi @VICTOR-CHUKWU how are you i try to go to that path but it works by my side i don't know why 
Screen Shot 2023-11-08 at 5 38 55 PM

**Review by Strangeal — APPROVED** [review]
Hi @ethred, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**